Update issue templates (#926)
DevExpress/devextreme-angular: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Report a bug in devextreme-angular
title: ''
labels: ''
assignees: ''

---

<!-- *************************************************************************************************

> To help us expedite the issue processing, please search opened and closed tickets in our support center (https://www.devexpress.com/Support/Center/Question/List/1) before you submit a new issue. The tickets can contain resolutions, workarounds, or information about the progress in fixing.

Reviewer: the issue processing?

Reviewer: the progress in fixing?

> To help us expedite the issue processing, please search opened and closed tickets in our support center (https://www.devexpress.com/Support/Center/Question/List/1) before you submit a new issue. The tickets can contain resolutions, workarounds, or information about the progress in fixing.

Reviewer 2: ```suggestion
To reduce the time it takes to process issues, search opened and closed tickets in our support center (https://www.devexpress.com/Support/Center/Question/List/1) before you submit a new issue. The tickets can contain resolutions, workarounds, or information about fixes.
```

You can also find information that can help you to resolve the issue in the following resources:

    - https://github.com/DevExpress/devextreme-angular/blob/master/README.md
    - https://js.devexpress.com/Documentation/Guide/Getting_Started/Widget_Basics_-_Angular/
******************************************************************************************************* -->

# Bug Report

<!-- Please provide the following information -->

**Package versions:**

    devexteme version:
    devextreme-angular version:

**Steps to reproduce:**

> <!-- Provide a sample application that illustrates the bug if possible. Do it using: 

Reviewer: Do it using?

Reviewer: Do not use 'using'

> <!-- Provide a sample application that illustrates the bug if possible. Do it using: 

Reviewer 2: ```suggestion
<!-- Provide a sample application that illustrates the bug if possible. You can use the following services: 
```
